### PR TITLE
Created and amended deltabug.txt

### PR DESCRIPTION
**Pull request by deltabug**

### CODE REVIEW

File: attendees/deltabug.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Hello,
+
+I am Deltabug. This is my first PyCon.
+I like Dragons
```
Comment (lgiordani) [inline attendees/deltabug.txt]: You should spell this with a lowercase "d" :wink:

**Review by lgiordani — CHANGES_REQUESTED** [review]

File: attendees/deltabug.txt (hunk 1 above)
Comment (deltabug) [reply attendees/deltabug.txt]: Hehe yes! In Twitter it's a capital "D" and I forget 😁

**Review by lgiordani — APPROVED** [review]

**Comment by lgiordani** [comment]
Well done! :+1:

**Comment by trallard** [comment]
🐉 we love dragons too! and 🦄

**Pull request merged**